src/api/config/initializers/influxdb_rails.rb

```diff
@@ -14,4 +14,13 @@
   config.use_ssl             = CONFIG['influxdb_ssl']
   config.time_precision      = CONFIG['influxdb_time_precision']
   config.series_name_for_sql = 'rails.sql'
+  config.tags_middleware = lambda do |tags|
+    # set a default tags
```

Comment:
_a default tags_? 

[Nitpick] To be honest, I don't think the comment is needed, I would remove it.